src/frontend/classic/App.svelte

```diff
@@ -96,70 +82,14 @@
     $: currentContent = selectedItem ? "viewer" : "home";
 </script>
 
-<Theme persist bind:theme>
-    <Header
-        id="Header"
-        expandedByDefault={false}
-        persistentHamburgerMenu={true}
-        company="HakuNeko"
-        platformName="Media & Anime - Downloader"
-        bind:isSideNavOpen
-    >
-        <div slot="skip-to-content">
-            <SkipToContent />
-        </div>
-        <HeaderUtilities>
-            <HeaderGlobalAction
-                on:click={() => win.minimize()}
-                aria-label="Minimize"
-                icon={MinimizeIcon}
-            />
-            <HeaderGlobalAction
-                on:click={() => (winMaximized ? win.restore() : win.maximize())}
-                aria-label="Maximize"
-                icon={winMaximized ? RestoreIcon : MaximizeIcon}
-            />
-            <HeaderGlobalAction
-                on:click={() => win.close()}
-                aria-label="Close"
-                icon={CloseIcon}
-            />
-            <HeaderAction bind:isOpen>
-                <HeaderPanelLinks>
-                    <HeaderPanelDivider>Interface</HeaderPanelDivider>
-                    <HeaderPanelLink>
```

Comment:
I'm sad that the UI settings aren't on the right side anymore, it was nice to test the UI changes. The menu is hiding everything. Could be in both places.

Reply:
I like to have one thing in one place I think it's easier but yes we can set it in two places for testing